scripts/analysis.py

```diff
@@ -1091,32 +1034,229 @@ def plot_power(cursor):
-
-
-        tailings = commodity_from_facility(cur, 'enrichment', ['tailings'])
-        stacked_bar_chart(tailings, timestep,
-                          'Year', 'Mass [MTHM]',
-                          'Tailings vs Time',
-                          'tailings',
+        plot_power(cur)
+        fuel_dict = fuel_usage_timeseries(cur, ['mox'])
+        stacked_bar_chart(fuel_dict, timestep,
+                          'Years', 'Mass[MTHM]',
+                          'Total Fuel Mass vs Time',
+                          'total_fuel',
                           init_year)
+        
+# combined case
+
+        # rep_dict = get_trade_dict(cur, 'separations', 'reactor', False, True)
+        # stacked_bar_chart(rep_dict, timestep,
+        #                  'Year', 'Mass [MTHM]',
+        #                  'reprocessing product vs time',
+        #                  'rep_product', init_year)
+        # tailings = commodity_in_out_facility(cur, 'enrichment',
+        #                                       ['tailings'], True)
+        # stacked_bar_chart(tailings, timestep,
+        #                  'Year', 'Mass [MTHM]',
+        #                  'Tailings vs Time',
+        #                  'tailings',
+        #                  init_year)
+        #fuel_dict = where_comm(cur, 'mox', ['uox_mixer', 'mox_mixer'])
+        # stacked_bar_chart(fuel_dict, timestep,
+        #                  'Years', 'Mass[MTHM]',
+        #                  'Total Fuel Mass vs Time',
+        #                  'total_fuel',
+        #                  init_year)
+        #demand = collections.OrderedDict()
+        #demand['pu_from_legacy'] = [i * .09 for i in fuel_dict['uox_mixer']]
+        #demand['pu_from_spent_mox'] = [i * .09 for i in fuel_dict['mox_mixer']]
+        # total_mox = ([x + y for x, y in zip(fuel_dict['uox_mixer'],
+        #                                    fuel_dict['mox_mixer'])])
+        #demand['pu_total'] = [i *.09 for i in total_mox]
+        #demand['tailings'] = [i * .91 for i in total_mox]
+        # multi_line_plot(demand, timestep,
+        #                'Years', 'Mass[MTHM]',
+        #                'Total Demand vs Time',
+        #                'demand',
+        #                init_year)
+
+        # reprocessing_waste = get_trade_dict(cur, 'separations',
+        #                                    'sink', False, False)
+        # stacked_bar_chart(reprocessing_waste, timestep,
+        #                  'Year', 'Mass [MTHM]',
+        #                  'reprocessing waste vs time',
+        #                  'repro_waste',
+        #                  init_year)
+
+        # plot_power(cur)
+        #dictionary = {}
+        #dictionary['uranium_utilization'] = u_util_calc(cur)
+        # stacked_bar_chart(dictionary, timestep,
+        #                  'Years', 'U Utilization Factor',
+        #                  'U Utilization vs Time',
+        #                  'u_util', init_year)
+
+"""
+"""
```

Comment:
We've removed all these lines and decided to store them in a separate file depending on the simulation.